openpype/hosts/max/plugins/publish/validate_frame_range_type.py

```diff
@@ -0,0 +1,32 @@
+import pyblish.api
+
+from pymxs import runtime as rt
+from openpype.pipeline.publish import (
+    RepairAction,
+    ValidateContentsOrder,
+    PublishValidationError
+)
+
+
+class ValidateFrameRangeType(pyblish.api.InstancePlugin):
+    """
+    Validates whether the User
+    specified Frame Range(Type 3) is used in render setting
+
+    """
+
+    label = "Validate Render Frame Range Type"
+    order = ValidateContentsOrder
+    families = ["maxrender"]
+    hosts = ["max"]
+    actions = [RepairAction]
+
+    def process(self, instance):
+        if rt.rendTimeType != 3:
+            raise PublishValidationError("Incorrect type of frame range"
+                                         " used in render setting..")
```

Comment:
The validation error should report what value is wrong and what value it should be set to. I feel even though this has a repair action that the report itself is still too vague.